src/libs/actions/Card.ts

```diff
@@ -0,0 +1,192 @@
+import Onyx, {OnyxUpdate} from 'react-native-onyx';
+import * as API from '@libs/API';
+import * as Localize from '@libs/Localize';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Response} from '@src/types/onyx';
+
+type Reason = 'damaged' | 'stolen';
```

Comment:
This name is not specific enough even considering it's not exported. Maybe `ReplacementReason`?

Reply:
@cubuspl42 Updated!